node/node/src/tests/runtime_api_mock.rs

```diff
@@ -114,7 +114,7 @@ sp_api::mock_impl_runtime_apis! {
 	}
 
 	impl sp_block_production_log::BlockProductionLogApi<Block, CommitteeMember<CrossChainPublic, SessionKeys>> for TestApi {
-		fn get_current_author() -> CommitteeMember<CrossChainPublic, SessionKeys> {
+		fn get_current_committee_member_for_slot(_slot: Slot) -> CommitteeMember<CrossChainPublic, SessionKeys> {
```

Comment:
I understand problem with "current" but we can make it `get_author`. There can be many committee member at given slot - basically all committee.